test/e2e/storage/testsuites/provisioning.go

```diff
@@ -195,6 +197,16 @@ func testProvisioning(input *provisioningTestInput) {
 		input.pvc.Spec.VolumeMode = &block
 		TestDynamicProvisioning(input.testCase, input.cs, input.pvc, input.sc)
 	})
+
+	It("should allow concurrent writes on the same node", func() {
```

Comment:
I don't think the test belongs in the provisioning suite. volumes might be a better fit? I guess it will need more boilerplate though since we can't piggyback on TestDynamicProvisioning.

Reply:
it might be okay if we make CapMultiPODs non-default though, then I would have to explicitly sign up my driver for this test anyway. Up to you..

Reply:
Yes, let's do that. My original reasoning was that all drivers must pass the test and therefore nothing special should be needed to run it, but as that's not the case, let's make it opt-in.

Reply:
Regarding where to put the test: I agree that this might belong into its own test suite, but it is not clear to me which one ("volume-features"?) and now is a bad time to introduce one because of the concurrent work on PR #72434. Let's keep it here, okay?

Reply:
agree